### PR TITLE
chore(stylelint): Update config to support new stylelint version

### PR DESCRIPTION
**Pull request by veekeys**
Updated configuration according to this https://github.com/stylelint/vscode-stylelint#%EF%B8%8F-only-css-and-postcss-are-validated-by-default

The only change I noticed was popover and somehow it was detecting inner rule to fail the class naming rule.. So moved disable to whole file, instead of every line.
From local testing seemed on save formatting was working in our scss files.

Good test would be:
Before pulling this branch update Stylelint extension in vscode to the newest version
Try messing up something in scss file and see if on save fixes it or at least the issue is marked (it should not)
Then pull the branch, run `yarn` and verify that it starts reacting to "wrong" code.

### CODE REVIEW

**Review by mayank99 — APPROVED** [review]
Works fine, but for some reason the stylelint extension v1.22.0 is working fine in main branch even without these changes.

Would be good if @bentleyvk or @FlyersPh9 or @adamhock could also verify whether 1.22.0 works on main branch without these changes.

If it does, I think it might have to do with dart sass.

File: yarn.lock [hunk 1]
```diff
@@ -3223,6 +3223,11 @@ postcss-scss@^4.0.2:
   resolved "https://registry.yarnpkg.com/postcss-scss/-/postcss-scss-4.0.3.tgz#36c23c19a804274e722e83a54d20b838ab4767ac"
   integrity sha512-j4KxzWovfdHsyxwl1BxkUal/O4uirvHgdzMKS1aWJBAV0qh2qj5qAZqpeBfVUYGWv+4iK9Az7SPyZ4fyNju1uA==
 
+postcss-scss@^4.0.3:
```
[the remark this answers is not in the document]
Comment (veekeys) [reply yarn.lock:3226]: for me it didnt and deleting lock file to create new one resulted in lots of changes...
If you or someone else could push this change, would be good

[the remark this answers is not in the document]
Comment (mayank99) [reply yarn.lock]: ok pushed

**Review by adamhock — COMMENTED** [review]
Running v1.22.0 of Stylelint on the main branch does not mark any styling issues, but after pulling this branch, 1.22.0 is catching styling errors

**Comment by bentleyvk** [comment]
> Running v1.22.0 of Stylelint on the main branch does not mark any styling issues, but after pulling this branch, 1.22.0 is catching styling errors

Same here.

**Pull request merged**